styleguide/derek/pattern-components/cust-story-river/_cust-story-river.scss

```diff
@@ -0,0 +1,163 @@
+// New cust story code
+.rsHomepage-customerStories-main-image {
+  background-image: url('../imgs/derek/banners/customer-story-background-480.jpg');
+  background-position: center;
+  background-size: cover;
+  display: none;
+  height: 450px;
+}
+
+.rsHomepage-customerStories-box {
+  background: $white;
+  display: block;
+  margin-bottom: 1px;
+  padding: 20px 50px 150px 20px;
+  text-decoration: none;
+}
+
+.cs-story-box0 {
+  @include make-sm-column(6);
+  @include make-md-column(3);
+}
+
+.cs-story-box1 {
+  @include make-sm-column(6);
+  @include make-md-column(3);
+  @include make-md-column-offset(6);
+}
+
+.cs-story-box2 {
+  @include make-md-column(3);
+  @include make-md-column-offset(3);
+  @include make-sm-column(6);
+}
+
+.cs-story-box3 {
+  @include make-md-column(3);
+  @include make-md-column-offset(3);
+  @include make-sm-column(6);
+}
+
+.rsHomepage-customerStories-wrapper {
+  background-image: url('../imgs/derek/banners/customer-story-background-480.jpg');
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  color: $black;
+  font-family: $heading;
+}
+
+.rsHomepage-customerStories-image {
+  margin-bottom: 20px;
+  width: 30%;
+}
+
+.rsHomepage-customerStories-header {
+  font-size: 4rem;
+  font-weight: lighter;
+  letter-spacing: .1em;
+  line-height: 1.25;
+}
+
+.rsHomepage-customerStories-text {
+  font-family: $copy;
+  font-size: 1.25rem;
+  line-height: 1.5rem;
+}
+
+.rsHomepage-customerStories-button {
+  border-radius: 3px;
+  color: $rackspace-red;
+  cursor: pointer;
+  font-weight: 400;
+  text-decoration: none;
+  width: 250px;
+}
+// < 480px
+@media screen and (max-width: $screen-xs) {
+  .rsHomepage-customerStories-main-image {
+    display: block;
+    height: 240px;
+  }
+
+  .rsHomepage-customerStories-bar {
+    border-top: 1px solid $gray-light;
+    padding-top: 10px;
+  }
+
+  .rsHomepage-customerStories-box {
+    margin-bottom: 0;
+    padding: 30px;
+  }
+}
+
+// 480px - 768
+@media screen and (max-width: $screen-sm) and (min-width: $screen-xs) {
+  .rsHomepage-customerStories-main-image {
+    display: block;
+    height: 380px;
+  }
+
+  .rsHomepage-customerStories-bar {
+    border-top: 1px solid $gray-light;
+    padding-top: 10px;
+  }
+
+  .rsHomepage-customerStories-box {
+    margin-bottom: 0;
+    padding: 30px;
+  }
+
+  .rsHomepage-customerStories-wrapper,
+  .rsHomepage-customerStories-main-image {
+    background-image: url('../imgs/derek/banners/customer-story-background-768.jpg');
+  }
+}
+// 768px - 991px
+@media screen and (max-width: $screen-sm-max) and (min-width: $screen-sm) {
+  .rsHomepage-customerStories-main-image {
+    display: block;
+  }
+
+  .rsHomepage-customerStories-wrapper {
+    background-image: none;
+    background-repeat: unset;
+    background-size: 0;
+  }
```

Comment:
Since we're relying on `.rsHomepage-customerStories-main-image` in this viewport, we should be safe to drop `.rsHomepage-customerStories-wrapper` here.

Reply:
^ I saw you removed all the other instances, we shouldn't need this one here either.